client/app/telecom/pack/slots/emailPro/add/pack-emailPro-add.html

```diff
@@ -0,0 +1,88 @@
+        <h1 data-translate="add_an_account"></h1>
+    </header>
+
+
+    <div class="telecom-section-content">
+
+        <toast-message></toast-message>
+        <div class="row widget-presentation">
+            <div class="col-md-6">
+                <h2 class="widget-presentation-title"
+                    data-translate="account_informations">
+                </h2>
+
+                <span data-translate="all_fields_are_required"></span>
+
+                <form name="accountForm"
+                      data-ng-submit="ctrl.add()">
+                    <div class="form-group">
+                        <div class="row">
+                            <div class="col-xs-12 input-select">
+                                <div>
+                                    <label class="control-label" data-translate="account_name"></label>
+                                    <div class="row">
+                                        <div class="col-xs-10">
+                                            <div class="input-select-container">
+                                                <div class="input-group">
+                                                    <input name="accountName"
+                                                        class="form-control"
+                                                        data-ng-model="ctrl.account.name"
+                                                        data-ng-model-options="{debounce: {'default': 800}}"
+                                                        pattern=".{5,}"
+                                                        title="{{ 'title_min_size' | translate }}"
+                                                        required
+                                                        maxlength="50"
+                                                        placeholder="{{ 'account_name' | translate }}">
+                                                    <span class="input-group-addon">@</span>
+                                                    <select name="accountDomain"
+                                                            class="form-control"
+                                                            data-ng-model="ctrl.account.domain"
+                                                            data-ng-options="item.value as item.label for item in ctrl.availablesDomains"
+                                                            required>
+                                                    </select>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="col-xs-2">
+                                            <h5 data-ng-show="accountForm.accountName.$dirty
+                                                        && accountForm.accountName.$valid
```

Comment:
I didn't reach any `<h3>` and `<h4>` tags. What's the reason of using an `<h5>` tag?